deploy/qdrant/templates/clusterdefinition.yaml

```diff
@@ -22,8 +22,14 @@ spec:
         builtIn: false
         exporterConfig:
           scrapePath: /metrics
-          scrapePort: 9187
+          scrapePort: 6333
```

Comment:
use port name instead

Reply:
check them all, including probes' ports

Reply:
Fixed, but scrapePort does not support port name, maybe we need to support it @yimeisun.